I0516 17:51:50.257747 140447003412288 run.py:28] Device: cuda
W0516 17:51:50.617945 140447003412288 utils.py:474] Not applying transforms to coordinates in unrecognized space 'ale_2mm'
I0516 17:51:50.703419 140447003412288 diagnostics.py:438] 7/214 coordinates fall outside of the mask. Removing them.
W0516 17:51:51.093531 140447003412288 utils.py:474] Not applying transforms to coordinates in unrecognized space 'ale_2mm'
I0516 17:51:51.783479 140447003412288 diagnostics.py:438] 91/2125 coordinates fall outside of the mask. Removing them.
W0516 17:51:52.111716 140447003412288 utils.py:474] Not applying transforms to coordinates in unrecognized space 'ale_2mm'
I0516 17:51:52.340945 140447003412288 diagnostics.py:438] 8/668 coordinates fall outside of the mask. Removing them.
W0516 17:51:52.717058 140447003412288 utils.py:474] Not applying transforms to coordinates in unrecognized space 'ale_2mm'
I0516 17:51:53.396851 140447003412288 diagnostics.py:438] 65/2098 coordinates fall outside of the mask. Removing them.
/well/nichols/users/pra123/anaconda3/envs/torch/lib/python3.11/site-packages/patsy/util.py:672: FutureWarning: is_categorical_dtype is deprecated and will be removed in a future version. Use isinstance(dtype, CategoricalDtype) instead
  return _pandas_is_categorical_dtype(dt)
/well/nichols/users/pra123/anaconda3/envs/torch/lib/python3.11/site-packages/patsy/util.py:672: FutureWarning: is_categorical_dtype is deprecated and will be removed in a future version. Use isinstance(dtype, CategoricalDtype) instead
  return _pandas_is_categorical_dtype(dt)
